app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Admin
+  class CategoriesController < Admin::BaseController
+    before_action :set_category, only: [:edit, :update, :destroy]
+
+    def index
+      @categories = Category.all.order(:id).includes(:slides)
+    end
+
+    def edit; end
+
+    def update
+      @category.assign_attributes(category_params)
+      if @category.update_attributes(category_params)
```

Comment:
__[RuboCop]__

Rails/ActiveRecordAliases: Use `update` instead of `update_attributes`.


<!-- Comment by [Sider](https://sider.review) -->